cmd/influxd/run.go

```diff
@@ -117,6 +117,17 @@ func Run(config *Config, join, version string, logWriter *os.File) (*messaging.B
 		}
 		log.Printf("data node #%d listening on %s", s.ID(), config.DataAddr())
 
+		// Start snapshot handler.
+		go func() {
```

Comment:
You can bind to `127.0.0.1:8086` and `:8086` and requests to `127.0.0.1:8086` will only go to the first listener. We should do a layered handler so we can multiplex like we do with server/broker but I wanted to get a simple implementation in first.